test/test_fx_experimental.py

```diff
@@ -595,6 +596,39 @@ def foo(x):
 
         traced = symbolic_trace_with_rewrite(foo)
 
+    def test_fetch(self):
+        attrs_for_lowering: Dict[Type, List[str]] = {
+            torch.nn.modules.conv.Conv2d: [
+                "weight", "bias", "kernel_size", "stride", "padding", "dilation", "groups", "padding_mode"
+            ],
+            torch.nn.modules.batchnorm.BatchNorm2d: ["weight", "bias", "running_mean", "running_var"],
+        }
+
+        class TestModule(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.conv = torch.nn.Conv2d(3, 3, 2)
+                self.bn = torch.nn.BatchNorm2d(3)
+
+            def forward(self, a):
+                a = self.conv(a)
+                a += a
+                return self.bn(a)
+
+        mod = TestModule()
+        traced = symbolic_trace(mod)
+        lift_lowering_attrs_to_nodes(traced)
+
+        for node in traced.graph.nodes:
+            if node.op == "call_module":
+                assert hasattr(node, "attrs_for_lowering")
+                para_list = attrs_for_lowering[locate(node.attrs_for_lowering["name"])]
```

Comment:
How about we just use string keys in `attrs_for_lowering` in this test instead of using `locate`?

Reply:
Sounds good!